app/controllers/calendars_controller.rb

```diff
@@ -3,22 +3,7 @@
 class CalendarsController < ApplicationController
   load_and_authorize_resource
 
-  # GET /calendars
-  # GET /calendars.json
-  def index
-    @national_calendars = Calendar.national.includes(:schools).order(:title)
-    @regional_calendars = Calendar.regional.includes(:schools).order(:title)
-    @school_calendars = Calendar.school.includes(:schools).order('schools.name')
-  end
-
   # GET /calendars/1
-  # GET /calendars/1.json
   def show
   end
```

Comment:
We had soft deletes of calendars, deleting has been disabled for now until we understand why it might need to happen, who might need to do it and what happens if we do